### PR TITLE
FIX #6091 Excess amount of image thumbnails in Financial area

### LINKED ISSUE #6091: Bug: Excess amount of image thumbnails in Financial area
# Bug

captura de pantalla 2016-12-08 a las 14 28 02

They are too much and distort the box

## Environment
- **Version**: 3.9.3
- **URL(s)**: /compta/index.php?mainmenu=accountancy&leftmenu=

### PR DESCRIPTION
**Pull request by marcosgdf**
FIX #6091 Excess amount of image thumbnails in Financial area

### CODE REVIEW

File: htdocs/core/class/html.formfile.class.php [hunk 1]
```diff
@@ -689,17 +689,22 @@ function showdocuments($modulepart,$modulesubdir,$filedir,$urlsource,$genallowed
...
     {
     	include_once DOL_DOCUMENT_ROOT.'/core/lib/files.lib.php';
 
     	$out='';
     	$this->infofiles=array('nboffiles'=>0,'extensions'=>array(),'files'=>array());
 
-		$file_list=dol_dir_list($filedir, 'files', 0, preg_quote(basename($modulesubdir),'/').'[^\-]+', '\.meta$|\.png$');	// Get list of files starting with name fo ref (but not followed by "-" to discard uploaded files)
+    	if (!$filter) {
+		    // Get list of files starting with name fo ref (but not followed by "-" to discard uploaded files)
+    		$filter = preg_quote(basename($modulesubdir),'/').'[^\-]+(odt|pdf)$';
```
Comment (eldy) [inline htdocs/core/class/html.formfile.class.php:704]: The submodule to generate documents may generate other files than .odt and .pdf but thay can also generate .txt, .odx, etc...
Any document must be returned, except ".meta" that is reserved to store a meta description file and images use to save thumbnails.

Comment (marcosgdf) [reply htdocs/core/class/html.formfile.class.php:704]: Please check the issue #6091 to see why that approach is not correct.

Comment (eldy) [reply htdocs/core/class/html.formfile.class.php:704]: The fix will gerenate new problems, by losing all files that are not .odt and .pdf.
Did you find a way to reproduce the bug ?

Comment (marcosgdf) [reply htdocs/core/class/html.formfile.class.php:704]: Yep, I did. I also showed it to @simnandez. Check the bug report because it is detailed there

**Comment by eldy** [comment]
It seems that we can't reproduct original bug #6091, so we should be able to cancel this PR.

**Pull request closed without merging**